schema/base.sql

```diff
@@ -3733,7 +3733,7 @@ ALTER TABLE ONLY selector
 --
 
 ALTER TABLE ONLY sso_session
-    ADD CONSTRAINT sso_session_pkey PRIMARY KEY (id, "timestamp");
+    ADD CONSTRAINT sso_session_pkey PRIMARY KEY (id);
```

Comment:
I guess it doesn't matter now that it's deployed to staging, but personally I prefer individual patch files using ALTER (let's ignore the fact that the patcher is bro-ken for now...) instead of modifying base.sql, for ease of deployment

Reply:
hmm. Fair.

Reply:
We really should get around to fixing the patcher - perhaps we can finally actually deploy the myury->myradio patch :p

Reply:
I'll open a ticket :)